VCNetworking/VCNetworking/encoders/PresentationResponseEncoder.swift

```diff
@@ -23,6 +23,28 @@ struct PresentationResponseEncoder: Encoding {
     }
     
     func encode(value: PresentationResponse) throws -> Data {
-        return try JSONEncoder().encode(value)
+        
+        let idTokenParam = "\(Constants.idToken)=\(try value.idToken.serialize())"
+        
+        guard let vpToken = value.vpToken else {
+            throw PresentationResponseEncoderError.noVerifiablePresentationInResponse
+        }
+        
+        let vpTokenParam = "\(Constants.vpToken)=\(try vpToken.serialize())"
+        
+        guard let state = value.state?.stringByAddingPercentEncodingForRFC3986() else {
+            throw PresentationResponseEncoderError.noStatePresentInResponse
+        }
+        
+        let stateParam = "\(Constants.state)=\(state)"
+        
+        let responseBody = "\(idTokenParam)&\(vpTokenParam)&\(stateParam)"
```

Comment:
I believe that state is optional, should we not set the entry if it's not there? (and I guess that would also impact L36). You can check with Kristina or Nithya to confirm if it's optional or not

Reply:
sounds good.